cfn-lint.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Would we be losing the "problem matcher" from https://github.com/ScottBrenner/cfn-lint-action/commit/4e7c712c4d5e271500b8ad3d714bf09eef6e5a58?

Reply:
Good question. I actually didn't know what this done. We could get this working again by simply adding the file back in and it would honour the files existence and the cfn-lint package would use it. However, I honestly didn't know what this was 🤔 @ScottBrenner could you explain? If it's needed all we need to do it re-commit it back in to the root of the action and copy it over to the $PATH I think 👍

Reply:
From https://github.com/actions/toolkit/blob/master/docs/problem-matchers.md
> Problem Matchers are a way to scan the output of actions for a specified regex pattern and surface that information prominently in the UI

This will display linting errors inline on pull requests, see the image in https://github.com/ScottBrenner/cfn-lint-action#github-action-for-cloudformation-linter for an example.

Might have to defer to original implementor @softprops regarding functionality, but in addition to this `cfn-lint.json` file we'll need something similar to https://github.com/ScottBrenner/cfn-lint-action/commit/4e7c712c4d5e271500b8ad3d714bf09eef6e5a58#diff-6f9d41d046756f0ddc2fcee0626bdb50100d12b88f293734eff742818e03efa2R7 looks like.